### PR TITLE
eos_lag_interfaces : Add unit testcases to eos_lag_interfaces

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
Added unit testcases to eos_lag_interfaces

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
lib/ansible/module_utils/network/eos/facts/lag_interfaces/lag_interfaces.py

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

File: test/units/modules/network/eos/test_eos_lag_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    # def test_eos_lag_interfaces_deleted(self):
+    #    set_module_args(dict(
+    #        config=[dict(
+    #            name="Port-Channel5",
+    #            members=[dict(
+    #               member="Ethernet1",
+    #            )]
+    #        )], state="deleted"
+    #    ))
+    #    commands=['interface Ethernet1', 'no channel-group']
```
Comment (Qalthos) [inline test/units/modules/network/eos/test_eos_lag_interfaces.py:164]:
```suggestion
```

File: test/units/modules/network/eos/test_eos_lag_interfaces.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                name="Port-Channel5",
+                members=[dict(
+                    member="Ethernet1",
+                    mode="on"
+                )]
+            )], state="replaced"
+        ))
+        self.execute_module(changed=False, commands=[])
+
+    # Bug #64615
```
Comment (Qalthos) [inline test/units/modules/network/eos/test_eos_lag_interfaces.py:127]: Is this still applicable?

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
Hi @GomathiselviS, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/arista.eos<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br><br>needs_info<br>

**Pull request closed without merging**